design-proposals/eim-cli.md

```diff
@@ -14,23 +14,22 @@ This design proposal will focus on the details required for the EIM support.
 - **Define** The user experience of the CLI tool should align with industry-standard tools such as kubectl, providing a familiar and intuitive interface. Since the Edge Infrastructure Manager (EIM) is primarily intended for customers to manage fleets of edge nodes, the CLI should be optimized for fleet management as the default use case. Managing individual edge nodes should be treated as an exception rather than the norm.
 - **Identify** common use cases and workflows.
   - Pre-registration of fleet of edge nodes (including Kubernetes)
```

Comment:
please list in 16 all the elements that we need with pre-registration with what will be mandatory by user and what will be optional
Mandatory to be filled: UUID, Serial, OS, Site, Region
Optional: 
- SB/FDE (False as default), 
- SSH key (no key)
- Metadata (none)
- Cloud-init(no file) 
- vPRO (false)
- k8s Cluster template (no template)